src/components/layouts/collection-page-layout.tsx

```diff
@@ -584,19 +668,45 @@ const CollectionPageLayout: React.FunctionComponent<CoursePageLayoutProps> = ({
                     >
                       <div
                         className="text-gray-600 dark:text-gray-300 flex flex-row items-center rounded hover:bg-gray-200 
-                      dark:hover:bg-gray-700 border-gray-300 bg-white dark:bg-gray-800 dark:border-gray-600 px-4 py-2 border transition-colors text-sm xs:text-base ease-in-out"
+                      dark:hover:bg-gray-700 border-gray-300 bg-white dark:bg-gray-800 dark:border-gray-600 px-4 py-2 border transition-colors text-sm xs:text-base ease-in-out shadow-sm"
                       >
                         <RSSIcon className="w-4 h-4 mr-1" /> RSS
                       </div>
                     </a>
                   </Link>
                 ) : (
-                  <div
-                    className="text-gray-600 dark:text-gray-300 flex flex-row items-center rounded hover:bg-gray-200 
-                  dark:hover:bg-gray-700 border-gray-300 bg-white dark:bg-gray-800 dark:border-gray-600 px-4 py-2 border transition-colors text-sm xs:text-base ease-in-out opacity-30"
+                  <a
+                    onClick={() => {
+                      track(`clicked disabled rss feed link`, {
+                        course: course.slug,
+                      })
+                      console.log('hello')
+                    }}
                   >
-                    <RSSIcon className="w-4 h-4 mr-1" /> RSS
-                  </div>
+                    <DialogButton
```

Comment:
suggestion [reuse]: this is the same component as the download button above with some text differences so I would recommend creating a component that allowed those strings to be supplied as properties